### PR TITLE
[AUTO] Support setting log_level from config

### PR DESCRIPTION
**Pull request by WeldonWangwang**
### Details:
 - *Support setting log_level from config*

### CODE REVIEW

**Review by yangwang201911 — APPROVED** [review]

File: src/plugins/auto/src/plugin.cpp [hunk 1]
```diff
@@ -360,6 +360,9 @@ IExecutableNetworkInternal::Ptr MultiDeviceInferencePlugin::LoadNetworkImpl(cons
     // updateFromMap will check config valid
     loadConfig.set_user_property(PreProcessConfig(config));
     loadConfig.apply_user_properties();
+    if (!setLogLevel(loadConfig.get_property(ov::log::level.name()).as<std::string>())) {
+        IE_THROW() << "Unsupported log level";
+    }
```
Comment (songbell) [inline src/plugins/auto/src/plugin.cpp:365]: then log from other calls except than loadmodel will still not getting outputs.

Comment (WeldonWangwang) [reply src/plugins/auto/src/plugin.cpp:367]: Added judgment on whether log_level comes from user settings.

File: src/plugins/auto/src/plugin_config.cpp [hunk 2]
```diff
@@ -51,6 +51,12 @@ void PluginConfig::set_property(const ov::AnyMap& properties) {
             internal_properties[name] = val;
             // when user call set_property to set some config to plugin, we also respect this and pass through the config in this case
             user_properties[name] = val;
+            if (kv.first == ov::log::level.name()) {
```
Comment (peterchen-intel) [inline src/plugins/auto/src/plugin_config.cpp:54]: Any test cases cover setting log level successfully and failed?

**Review by wangleis — APPROVED** [review]

File: src/plugins/auto/src/plugin_config.cpp (hunk 2 above)
Comment (WeldonWangwang) [reply src/plugins/auto/src/plugin_config.cpp:54]: Added test case.

**Review by songbell — APPROVED** [review]

**Review by wgzintel — APPROVED** [review]

**Pull request merged**